<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMainMenu</class>
 <widget class="QWidget" name="FrmMainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 127, 255), stop:1 rgba(0, 255, 255, 255));</string>
  </property>
  <widget class="QStackedWidget" name="mainStackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0)</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btnOpenAdministration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0.5);
border: 2px solid rgba(0,0,0,0);
border-radius: 15px;
font-size: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>Administration</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font-size: 30px;
font-weight: 400;
color: white;</string>
     </property>
     <property name="text">
      <string>Picadvisor Logo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenHelp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0.5);
border: 2px solid rgba(0,0,0,0);
border-radius: 15px;
font-size: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>Need help?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenPlayModal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0.5);
border: 2px solid rgba(0,0,0,0);
border-radius: 15px;
font-size: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>Play Now!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font-size: 35px;
font-weight: 400;
color: white;</string>
     </property>
     <property name="text">
      <string>Menu Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogout">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(242, 80, 60, 0.8);
border: 2px solid rgba(0,0,0,0);
border-radius: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>Logout here</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>190</y>
       <width>331</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0.5);
font-size: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>110</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font-size: 30px;
font-weight: 400;
color: white;</string>
     </property>
     <property name="text">
      <string>Highscores</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>163</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0.5);
border: 2px solid rgba(0,0,0,0);
font-size: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font-size: 30px;
font-weight: 400;
color: white;</string>
     </property>
     <property name="text">
      <string>Picadvisor Logo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGoMenu">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255,255,255,0.5);
border: 2px solid rgba(0,0,0,0);
border-radius: 15px;
font-size: 15px;
font-weight: 500;
padding: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt; Main Menu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font-size: 35px;
font-weight: 400;
color: white;</string>
     </property>
     <property name="text">
      <string>How to play?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbHowToPlay">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>411</width>
       <height>271</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Lorem Ipsum dolor sit amet consectetur, sadipcising en solus damur kanvetus en radon padros. Superioris canletvitis da pantor gigas. Superioris canletvitis da pantor gigas. Superioris canletvitis da pantor gigas. Lorem Ipsum dolor sit amet consectetur, sadipcising en solus damur kanvetus en radon padros. Superioris canletvitis da pantor gigas. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbHowToPlay_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>250</y>
       <width>411</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Lorem Ipsum dolor sit amet consectetur, sadipcising en solus damur kanvetus en radon padros. Superioris canletvitis da pantor gigas. Superioris canletvitis da pantor gigas. Superioris canletvitis da pantor gigas. Lorem Ipsum dolor sit amet consectetur, sadipcising en solus damur kanvetus en radon padros. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>200</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
font-size: 35px;
font-weight: 400;
color: white;</string>
     </property>
     <property name="text">
      <string>About the modes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
